src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/wiremock/NonAssociationsApiMockServer.kt

```diff
@@ -35,7 +35,7 @@ class NonAssociationsApiExtension : BeforeAllCallback, AfterAllCallback, BeforeE
 
 class NonAssociationsApiMockServer : WireMockServer(WIREMOCK_PORT) {
   companion object {
-    private const val WIREMOCK_PORT = 8091
+    private const val WIREMOCK_PORT = 8092
   }
```

Comment:
this was conflicting with an IntelliJ opened port - but just for me - so I have shifted up by one in frustration